book/internals.rst

```diff
@@ -238,8 +238,8 @@ add the following code at the beginning of your listener method::
 
 .. tip::
 
-    If you are not yet familiar with the Symfony EventDispatcher, read the
-    :doc:`EventDispatcher component documentation </components/event_dispatcher/introduction>`
+    If you are not yet familiar with the Symfony EventDispatcher component,
```

Comment:
for example here we use
VarDumper component https://github.com/symfony/symfony-docs/blame/4dd1f25090fd2c5eb1f6ec2a6c2c1613bef1c856/changelog.rst#L276
think its much better than "var dumper component" 
https://github.com/symfony/symfony-docs/blame/4dd1f25090fd2c5eb1f6ec2a6c2c1613bef1c856/changelog.rst#L475

so i am :+1: